test/not_loaded_script_test.html

```diff
@@ -0,0 +1,10 @@
+<!DOCTYPE html>
```

Comment:
See https://github.com/firebase/firebase-dart/commit/529f33057f376ae5fbf49b57d39fb7b4383e22bd

Seems that we don't handle this consistently between Dartium and dart2js – if you're bored you could file an issue on the SDK for this...